hrjobcontract/js/src/job-contract/controllers/form/form-general.js

```diff
@@ -89,7 +89,7 @@ define([
         months = m.diff(dateStart, 'months');
 
         m.add(-months, 'months');
-        days = m.diff(dateStart, 'days');
+        days = m.diff(dateStart, 'days') + 1;
```

Comment:
I was still thinking about this and wondering if this whole method could be implemented in a better way. Maybe [Moment.js Humanize](http://momentjs.com/docs/#/durations/humanize/) can help with that?